test/cli/run.spec.ts

```diff
@@ -89,24 +89,24 @@ ${str}`);
   }
 }
 
-type Options = {
+interface Options {
   args?: string[];
   encoding?: BufferEncoding;
-  env?: Record<string, string>;
```

Comment:
I didn't have  strong preference for `Record` vs `{[index]}`, and we use the latter more often.